### ISSUE TITLE
The ability to specify whether the text should be listed in RSS/OPML

### ISSUE THREAD

**Issue by rojenzaman** [melmothx/amusewiki#404]
I have some special pages (/special) that are updated frequently. Each update the texts appear in the RSS feed. Can we specify if text is included in the RSS feed? Like this:

```
#rssindex no
```

or

```
#siteindex no
```

This feature can avoid confusion for site users.

**Comment by rojenzaman** [comment]
I tried update `pubdate` to very an old date. But wiki engine assumes the real creation time when creating feed.

**Comment by melmothx** [comment]
I think the problem is here:

https://github.com/melmothx/amusewiki/blob/master/lib/AmuseWikiFarm/Schema/Result/Site.pm#L4736

I honesty don't recall why for special pages the pubdate is ignored and the file timestamp is used, so I think the pubdate at this point should be honored. The automatic pubdate is used if there is no `#pubdate` anyway.

**Comment by melmothx** [comment]
@rojenzaman do you have the chance to apply https://github.com/melmothx/amusewiki/commit/d3d4b14197d5d4dfc443a7b245d7783bd89f3390 and see if that's enough?

**Comment by rojenzaman** [comment]
Thank you! I tried, that's enough.

**Comment by rojenzaman** [comment]
Before closing this issue, there is one more RSS related question. The RSS attachment file type is epub, any chance to remove it or change it to PDF or specify what should or shouldn't be included? Here the line: https://github.com/melmothx/amusewiki/blob/df5585bc97ffc9eab069bdc4d9aedd5bad7a0d98/lib/AmuseWikiFarm/Schema/Result/Site.pm#L4763-#L4772

In OPDS the default file type was epub but I was able to fix this issue with nginx sub_filter: 

```nginx
sub_filter '.epub\" type=\"application/epub+zip\"' '.pdf\" type=\"application/pdf\"';
```

However, I cannot change RSS  enclosure file type with sub_filter because there is a length required attribute in the enclosure element.

Do you think this is a difficult or unnecessary feature? Would it be ok for you to adapt it to Amusewiki settings? Or should I create a new issue about it?

**Comment by melmothx** [comment]
@rojenzaman new issue please. There is some logic to hammer out. The epub was chosen because it's always generated. That's not the case with PDF (we have multiple ones). Even an attachment could be a candidate. So I think the request needs to be elaborated a bit. There could be a setting to choose between: "stock epub" (default), "highest priority generated file", "first attachment or highest priority generated file" something like that.

**Comment by melmothx** [comment]
Released in 2.560
